src/common/id/SegmentId.h

```diff
@@ -0,0 +1,81 @@
+/* Copyright (c) 2021 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License.
+ */
+
+#ifndef COMMON_ID_SEGMENTINCR_H_
+#define COMMON_ID_SEGMENTINCR_H_
+
+#include <double-conversion/utils.h>
+
+#include "clients/meta/MetaClient.h"
+#include "common/base/Base.h"
+
+namespace nebula {
+// Segment auto-increase id
+class SegmentId {
+  FRIEND_TEST(SegmentIdTest, TestConcurrency);
+
+ public:
+  static SegmentId& getInstance() {
+    ASSERT(client_ != nullptr);
+    ASSERT(runner_ != nullptr);
+    static SegmentId instance;
+    return instance;
+  }
+
+  ~SegmentId() = default;
+
+  SegmentId(const SegmentId&) = delete;
+
+  SegmentId& operator=(const SegmentId&) = delete;
+
+  Status init(int64_t step) {
+    step_ = step;
+    if (step < 10000) {
```

Comment:
```
constexpr kMinSegmentSize = 10000;
```
10000 might be too small, we need to meet the base requirements. I sugguest we have ten minutes allowance:
max insert/secs  * 600. Mignt be 400000 * 600 = 240000000